### DISCUSSION TITLE
Docker Container creation fails

### DISCUSSION THREAD

**Discussion by KingKahn123** [camueller/SmartApplianceEnabler#356] [Installation und Betrieb]
Hi,

I would like to migrate my existing SAE installation to docker. However when I am trying to start the container with docker-compose I get following error message:

"Cannot start service sae: failed to create the macvlan port: operation not supported"

I have adapted the yaml file with my network settings. Any guess what the problem is?

**Comment by camueller** [comment]
Perhaps this helps: https://www.reddit.com/r/selfhosted/comments/rdjlup/docker_error_response_from_daemon_failed_to/

The problem is most likely related to your specific platform or network setup. Therefore I doubt that somebody will the solution to your problem.

I'm running SAE in a docker container using `docker compose` using the configuration files contained in the Github repository.

**Reply by KingKahn123** [reply]
Ok understood... But I am running no Ubuntu on the raspberry. But then I need to search... how to fix this problem for me.

**Reply by camueller** [reply]
I'm running Raspbian Buster with Docker.

**Reply by KingKahn123** [reply]
yes correct. Same for me... But the solution in the link above is for Raspbians with Ubuntu. Therefore I cannot install the suggested package.

**Reply by camueller** [reply]
If you run the same software stack like I do this leaves your network configuration as the only source of the problem.

**Reply by KingKahn123** [reply]
Yes... I will dig into it. I will give you an update if I've found a solution.
